Automattic/jetpack
# Newsletter: move subscription settings to their own section

As we are moving Newsletter settings to their own section in Calypso for WP.com sites (https://github.com/Automattic/wp-calypso/pull/79986) to improve discoverability, I'd like to do the same for Jetpack settings.


## Proposed changes:

* Add new "newsletter" section in settings
* Move subscription settings from "Writing" to the new section

<img width="1453" alt="image" src="https://github.com/Automattic/jetpack/assets/87168/804c98ca-a6c8-40c9-bff6-a5d2e90bdd31">


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to "Jetpack settings → Newsletter"
* Ensure all toggles work
* Go to "Discussion" settings and confirm newsletter settings aren't there anymore
